pkg/apis/core/types.go

```diff
@@ -544,6 +566,41 @@ const (
 	PersistentVolumeClaimNodeResizeFailed ClaimResourceStatus = "NodeResizeFailed"
 )
 
+// +enum
+type PersistentVolumeClaimModifyVolumeStatus string
+
+const (
+	// Pending indicates that the PersistentVolumeClaim cannot be modified due to requirements not being met, such as
+	// the specified VolumeAttributesClass doesn't exist
+	PersistentVolumeClaimControllerModifyVolumePending PersistentVolumeClaimModifyVolumeStatus = "ControllerModifyVolumePending"
```

Comment:
The "ControllerModifyVolume" prefixes to all these status values is redundant, as this field is already under a "ModifyVolumeStatus" struct.

Reply:
If we are removing this struct, then we keep this?